src/main/resources/templates/kotlin-lang/kotlinClassGraphqlResponseProjection.ftl

```diff
@@ -26,10 +26,12 @@ import java.util.Objects
 </#list>
 open class ${className} : GraphQLResponseProjection() {
 
-<#if fields?has_content>
-    override fun `all$`(): ${className} = `all$`(${responseProjectionMaxDepth})
+<#if fields?has_content && generateAllMethodInProjection>
+    var projectionDepthOnFields: MutableMap<String, Int> = mutableMapOf()
```

Comment:
```
private val projectionDepthOnFields: MutableMap<String, Int> = mutableMapOf()
```

Reply:
Nice suggestion, but I think it should be:

```kotlin
    private final val projectionDepthOnFields: MutableMap<String, Int> by mutableMapOf()
````